### PR TITLE
Update API list variables example.

### PR DESCRIPTION
**Pull request by felicianotech**

### CODE REVIEW

File: jekyll/_data/api.yml [hunk 1]
```diff
@@ -498,8 +498,8 @@ list_environment_variables:
   try_it: false
   url: "/api/v1.1/project/:vcs-type/:username/:project/envvar"
   method: "GET"
-  description: "Lists the environment variables for :project"
-  response: '[{"name":"foo","value":"xxxx"}]'
+  description: "Lists the environment variables for :project. For security reasons, only four 'x's and the last 4 characters of the value will be returned. This is the same as viewing existing environment variables via the CircleCI webapp."
```
Comment (michelle-luna) [inline jekyll/_data/api.yml]: Thanks, for clarity, suggest rewriting this to: Returns four `x` characters plus the last four alphanumeric characters of the value, consistent with the display of environment variable values in the CircleCI application.

Comment (felicianotech) [inline jekyll/_data/api.yml]: I'm fine with this except "alphanumeric" isn't correct. It'll return any supported characters. I'm not 100% sure that those are but I strongly think it's any ASCII characters.

You okay with "alphanumeric" -> "ASCII" ?

Comment (michelle-luna) [inline jekyll/_data/api.yml]: Yep, works for me, thanks!

Comment (felicianotech) [inline jekyll/_data/api.yml]: Done

**Comment by michelle-luna** [comment]
LGTM

**Pull request merged**